Add mobile menu features

Diana-Duya opened the pull request:
In this PR:

- I opened a new branch  and modified the HTML and CSS files
- I also added a JavaScript file to the root directory and linked it to HTML file.
- i set up linters locally and fixed linter errors.
- I then committed the changes.

Diana-Duya committed:
Add-mobile

HAMMAS-SALEEM commented on index.html:
```diff
@@ -16,13 +17,17 @@
-                <li><a href=""><img src="image/Union.png" alt="union-icon" class="union"></a></li>
-            </ul>         
-            <a class="hambuger" href=""><img src="image/Union.svg" alt="union-icon" ></a>            
+            <ul class="nav-menu">
+                <li> <a href="#link1" class="nav-item">Portoflio</a></li> 
+                <li> <a href="#about" class="nav-item">About</a></li>
+                <li> <a href="#link2" class="nav-item">Contact</a></li>
+                <li><a href="#link2"><img src="image/Union.png" alt="union-icon" class="union"></a></li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar bar1"></span> 
+                <span class="bar bar2"></span>
+                <span class="bar bar3"></span> 
+            </div>         
```
### Your Mobile Menu
![student-mobile-menu (copy)](https://user-images.githubusercontent.com/93039359/189873251-fca22391-bf5f-4000-b7bc-efd63cc49d0c.png)

### Figma Mobile Menu
![figma-mobile-menu (copy)](https://user-images.githubusercontent.com/93039359/189873269-56c7a750-dca6-4967-825e-cbef8d73cf9b.png)

- [ ] You can see the difference between your mobile menu and the Figma mobile menu. I have added the changes in the screenshots. Please compare both screenshots and make the changes :blush:

HAMMAS-SALEEM reviewed (changes requested):
# Required Changes :recycle:
![almost-there](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Hey @Diana-Duya :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional
**[Optional]**
- [ ] Your branch name is not descriptive enough. You are using **New-mobile**, you could have used **feature/add-menu** which is more descriptive and tells what is happening in this branch :thinking:

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Diana-Duya committed:
Changes to mobile-menu

Diana-Duya committed:
fix linter error

Diana-Duya commented:
Hello @HAMMAS-SALEEM, 
I implemented all the changes ;

- Expanded the nav-menu
- Added the x button to my design 
- Tried to replicate the figma design as much as possible

BertrandConxy commented on index.html:
```diff
@@ -16,13 +17,20 @@
-            </ul>         
-            <a class="hambuger" href=""><img src="image/Union.svg" alt="union-icon" ></a>            
+            <ul class="nav-menu">
+                <li> <a href="#link1" class="nav-item">Portoflio</a></li> 
```
- [ ] [OPTIONAL] Good job so far 👏. I think there is a typo here on the Portfolio. Please check it out again 😃 👍 
![image](https://user-images.githubusercontent.com/90222110/189949670-d879609f-46fd-4ed0-848f-487a811b0a5d.png)

BertrandConxy reviewed (approved):
Hi @Diana-Duya , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
